apis/v1alpha1/generator.yaml

```diff
@@ -1,11 +1,51 @@
 ignore:
   resource_names:
-    - Snapshot
     - User
     - ACL
     - Cluster
     - SubnetGroup
+  field_paths:
+    - DescribeSnapshotsInput.ClusterName
+    - DescribeSnapshotsInput.Source
 resources:
+  Snapshot:
+    exceptions:
+      errors:
+        404:
+          code: SnapshotNotFoundFault
+      terminal_codes:
+        -  InvalidParameterCombinationException
+        -  InvalidSnapshotStateFault
+        -  SnapshotAlreadyExistsFault
+        -  SnapshotQuotaExceededFault
+        -  TagQuotaPerResourceExceeded
+        -  ClusterNotFoundFault
+        -  InvalidClusterStateFault
```

Comment:
Maybe we should remove this one as well, if there is a chance the cluster will recover into a good state.